portfolio/src/main/webapp/style.css

```diff
@@ -389,6 +394,16 @@ table {
 }
 /*End charts*/
 
+/*Start map*/
+#map-div {
+  border: none;
+  margin-right: auto;
+  margin-left: auto;
```

Comment:
Nit, but the Google style guide recommends combining all of these into a single line (i.e. margin: 0 auto 0 auto)

Reply:
Wow, I don't know that! Fixing my code right now...

Reply:
fixed!